auth-web/src/composables/transactions-factory.ts

```diff
@@ -0,0 +1,76 @@
+import { Transaction, TransactionFilterParams, TransactionState } from '@/models/transaction'
+import { computed, reactive, watch } from '@vue/composition-api'
+import { Organization } from '@/models/Organization'
+import PaymentService from '@/services/payment.services'
+import { useStore } from 'vuex-composition-helpers'
+
+const transactions = (reactive({
+  filters: {
+    isActive: false,
+    filterPayload: {
+      dateFilter: {
+        startDate: '',
+        endDate: ''
+      }
+    },
+    pageLimit: 0,
+    pageNumber: 1
+  } as TransactionFilterParams,
+  loading: false,
+  results: [] as Transaction[],
+  totalResults: 0
+}) as unknown) as TransactionState
+
+export const useTransactions = () => {
+  const store = useStore()
+  const currentOrganization = computed(() => store.state.org.currentOrganization as Organization)
+
+  const loadTransactionList = async (filterField?: string, value?: any) => {
+    transactions.loading = true
+    if (filterField) transactions.filters.filterPayload[filterField] = value
+    let filtersActive = false
+    for (const i in transactions.filters.filterPayload) {
+      if (i === 'dateFilter') {
```

Comment:
rename i to payload or key? Usually when you think of i.. it's an integer for an index